Makefile

```diff
@@ -1,22 +1,40 @@
 
 NAME = floop
 FILES = cmd/main.go cmd/cli.go
+BRANCH = $(shell git rev-parse --abbrev-ref HEAD)
+COMMIT = $(shell git rev-parse --short HEAD)
+VERSION = $(shell git describe | sed -e "s/^v//")
```

Comment:
Understood.  I added an initial tag on the repo so the condition where no tag is present should never happen from now on.